src/systems/sound-effects-system.js

```diff
@@ -148,6 +149,9 @@ export class SoundEffectsSystem {
       : new THREE.PositionalAudio(this.scene.audioListener);
     positionalAudio.setBuffer(audioBuffer);
     positionalAudio.loop = loop;
+    if (!disablePositionalAudio) {
+      positionalAudio.panner.panningModel = getPanningModelType();
```

Comment:
If you make the change I recommend in the other comment, be careful that `getPanningModelType` could be `null`.